geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/build

```diff
@@ -124,7 +134,7 @@ def main() -> None:
         if not args.no_pull:
             # Pull all the images
             if not args.service:
-                run(args, ["docker-compose", "pull"], check=True)  # nosec
+                run(args, ["docker-compose", "pull"], exit_on_error=True)  # nosec
```

Comment:
We may have errors in docker-compose pull with locally builded images.

Reply:
Yes, and?

Reply:
If we exit on error, it not possible anymore to complete build without --no-pull.

But it seems this has been fixed here : https://github.com/camptocamp/c2cgeoportal/pull/10343/files

So it seems ok now.

Reply:
Done :-)